heuristic/alns.py

```diff
@@ -280,15 +329,7 @@ def calculate_objective(self, state, destroy, repair):
             self.demand,
             destroy_repair_set,
         )
-        delta_calculate_deviation_from_contracted_hours(
-            state,
-            employees,
-            self.contracted_hours,
-            self.weeks,
-            self.time_periods_in_week,
-            self.competencies,
-            self.time_step,
-        )
```

Comment:
Same as below. Is this replaced with a new function. If not why is it removed?

Reply:
Nice! Looking into som time-related bugs right now, but will fix this afterwards

Reply:
I see that this function shouldn't run. My mistake

Reply:
If it isn't already changed to delta_calculate_negative_deviation_from_contracted_hours

Reply:
Should this be re-introduced or not?

Reply:
There should be a calculation over deviation from contracted hours. I think the correct function is delta_calculate_negative_deviation_from_contracted_hours. If this is not present in the function it should be included.